src/resolve/resolved-manager.c

```diff
@@ -1054,6 +1054,68 @@ static int manager_ipv6_send(
         return sendmsg_loop(fd, &mh, 0);
 }
 
+int send_dns_notification(Manager *m, DnsAnswer *answer, const char *query_name)
+{
```

Comment:
Coding style: move `{` at the end of the previous line: `int send_dns_notification(...) {`

Reply:
Fixed in latest patch